client/src/components/R&R/ReviewsList.jsx

```diff
@@ -9,16 +9,35 @@ const ReviewsList = () => {
     reviewSort,
     setReviewSort,
   } = useContext(ProductContext);
+  const [updateCount, setUpdateCount] = useState(0);
+  const myStorage = window.localStorage;
+  if (!myStorage.getItem('listCount')) {
+    myStorage.setItem('listCount', reviews.results.length);
+  }
+
+  const decreaseListItemsCount = () => {
+    myStorage.setItem('listCount', myStorage.getItem('listCount') - 1);
+    setUpdateCount(Math.random());
+  };
+
 
   return (
     <div className="reviewsList">
       <div className="numberAndSortType">
-        Showing {reviews.count} of {' '}
-        {Number(reviewsMeta.recommended.true) + Number(reviewsMeta.recommended.false) + ''} {' '}
+        Showing {myStorage.getItem('listCount')} of {' '}
+        {
+          Number(reviewsMeta.recommended.true ?
+            reviewsMeta.recommended.true : 0) +
+          Number(reviewsMeta.recommended.false ?
+            reviewsMeta.recommended.false : 0) + ''} {' '}
```

Comment:
It took me awhile to understand what you're doing here. I'd suggest trying to break things up across a few variables to make things more readable.